src/Components/ServiceNotConfigured/ServiceNotConfigured.js

```diff
@@ -1,15 +1,51 @@
-import { Bullseye, EmptyState, EmptyStateIcon, Title } from '@patternfly/react-core';
+import {
+    Bullseye, EmptyState, EmptyStateBody,
+    EmptyStateIcon, Title, Stack,
+    StackItem, ClipboardCopy, Button
+} from '@patternfly/react-core';
 import { WrenchIcon } from '@patternfly/react-icons';
 import React from 'react';
 import './ServiceNotConfigured.scss';
 
 export const ServiceNotConfigured = () => (
     <Bullseye>
-        <EmptyState className='empty-state-not-configured'>
+        <EmptyState className="empty-state-not-configured">
             <EmptyStateIcon icon={WrenchIcon} />
             <Title headingLevel="h5" size="lg">
-                Resource optimization is not yet configured
+              Resource optimization isn&apos;t configured yet
             </Title>
+            <EmptyStateBody>
+                <Bullseye>
+                    <Stack hasGutter>
+                        <StackItem>To start configuration, follow these steps.</StackItem>
+                        <StackItem>
+                            1. Prerequisites: insights-client, ansible (Read FAQ if you
+                don&apos;t know how to install it).
+                        </StackItem>
+                        <StackItem>
+                            2. Download and run playbook.
+                        </StackItem>
+                        <StackItem>
+                            <ClipboardCopy>
+                                wget https://raw.githubusercontent.com/RedHatInsights/ros-backend/main/ansible-playbooks/ros_install_and_set_up.yml
```

Comment:
"wget" is not installed by default, we should use "curl -O" 

```suggestion
                                curl -O https://raw.githubusercontent.com/RedHatInsights/ros-backend/main/ansible-playbooks/ros_install_and_set_up.yml
```